vpc_endpoint_policies/ec2_endpoint_policy.json

```diff
@@ -46,6 +46,26 @@
                 }
             }
         },
+        {
+            "Sid": "AllowExternalAMIForMarketplace",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "*"
+            },
+            "Action": [
+                "ec2:RunInstances"
+            ],
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "aws:PrincipalOrgID": "<my-org-id>",
+                    "aws:ResourceAccount": "<external-account-id>"
+                },
+                "Null": {
+                    "ec2:ImageId": "false"
```

Comment:
For this I would prescirbe the use of ec2:Owner condition. I would recommend that there be a second statement that's an allow but allows "StringEquals" {"ec2:Owner" : "aws-marketplace"} to allow an ami from the marketplace, but also not invalidate the dataperimeter of only normally allowing "only your organizations" AMIs.

Becuase we expect people to copy these examples and apply them mostly as-is with fill in the blanks, in the repo we'll improve the wording to indicate support for marketplace ami's if that is desired.